lamindb_setup/core/_hub_core.py

```diff
@@ -253,29 +258,39 @@ def _delete_instance(
     return None
 
 
-def delete_instance_record(
-    instance_id: UUID,
-) -> None:
+def delete_instance_record(instance_id: UUID, access_token: str | None = None) -> None:
     return call_with_fallback_auth(
-        _delete_instance_record,
-        instance_id=instance_id,
+        _delete_instance_record, instance_id=instance_id, access_token=access_token
     )
 
 
-def init_instance(isettings: InstanceSettings) -> None:
-    return call_with_fallback_auth(_init_instance, isettings=isettings)
+def init_instance(
+    isettings: InstanceSettings,
+    account_id: str | None = None,
+    access_token: str | None = None,
+) -> None:
+    return call_with_fallback_auth(
+        _init_instance,
+        isettings=isettings,
+        account_id=account_id,
+        access_token=access_token,
+    )
 
 
-def _init_instance(isettings: InstanceSettings, client: Client) -> None:
+def _init_instance(
+    client: Client, isettings: InstanceSettings, account_id: str | None = None
```

Comment:
Well, I guess it works if `client` is the first argument. Then let's keep it this way.

But `account_id` has to be a `UUID` not a `str`!

Reply:
done